spec/requests/registrations_spec.rb

```diff
@@ -179,6 +179,17 @@ def mock_recaptcha_verification
         expect(User.all.size).to be 1
       end
 
+      it "marks as registerd" do
+        post "/users", params:
```

Comment:
**suggestion (non-blocking):** Rewrite this a change assertion so we really see that `registered_at` changes from `nil` to something:

```ruby
expect do
  post "/users", params: ...
end.to change { User.last.registered_at }
expect(User.last.registered).to be true
```